config/software/ruby.rb

```diff
@@ -128,6 +128,14 @@
     patch source: "ruby-no-stack-protector.patch", plevel: 1, env: patch_env
   end
 
+  # accellerate requires of c-extension.
```

Comment:
```suggestion
  # accelerate requires of c-extension.
```

Reply:
that has been one of my favorite typos for 30 years at least